### ISSUE TITLE
Meshes rendering in the wrong orientation

### ISSUE THREAD

**Issue by civerachb-cpr** [ros-visualization/rviz#1688]
It appears that there's a bug in Rviz 1.13.22 where links with STL or DAE visuals are being rendered incorrectly  The underlying links are correctly located, but the visual and collision geometry is a mess.

For example, here is the [Clearpath Warthog](https://github.com/warthog-cpr/warthog) model viewed using the command `roslaunch warthog_viz view_robot.launch`:

Rviz 1.13.18 | Rviz 1.13.22 | Gazebo 9.19.0 |
|-----------------|------------------|---------------------|
| ![warthog-correct](https://user-images.githubusercontent.com/59611394/146228563-67ae0f6c-d983-4130-aa89-b2b4c3fec31f.png) | ![warthog-chris](https://user-images.githubusercontent.com/59611394/146228648-dbb93702-958c-4ce2-812d-8b3f72da9fee.png) | ![warthog-gazebo](https://user-images.githubusercontent.com/59611394/146229283-5bd2cc78-3496-46fb-9ae3-1f6b8f6de1d7.png) |

Adding the RobotModel vizualization to Rviz results in the same mal-formed visual as above; the meshes are all oriented incorrectly in Rviz, even though Gazebo is publishing the TF and joint states correctly.

This also affects simpler models, like the [VLP-16 from velodyne_description](http://wiki.ros.org/velodyne_description)
![dingo_velodyne_tf](https://user-images.githubusercontent.com/59611394/146228815-1692c2d8-fd9c-400d-82fd-9571f12eb9ba.png)

You can see the Velodyne's base link and sensor origin frames are oriented vertically relative to each other, but the mesh itself is rotated off to the side.  The above was created with this included in the URDF for the robot:

```
  <xacro:include filename="$(find velodyne_description)/urdf/VLP-16.urdf.xacro" />
  <xacro:VLP-16 parent="front_mount" topic="velodyne_points">
    <origin xyz="0 0 0" rpy="0 0 0" />
  </xacro:VLP-16>
```

### environment
* OS Version: Ubuntu 18.04
* ROS Distro: Melodic
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
    ```
    [ INFO] [1639587205.192206801]: rviz version 1.13.22
    [ INFO] [1639587205.192246967]: compiled against Qt version 5.9.5
    [ INFO] [1639587205.192257016]: compiled against OGRE version 1.9.0 (Ghadamon)
    ```
* If source build, which git commit? n/a -- installed via .deb from official ROS apt sources
* System locale, i.e. the output of `echo "$LANG $LC_NUMERIC"`: `en_US.UTF8`

_Before reporting a rendering issue, try running RViz with `LANG=C rviz`!_
I tried this, the result was the same; the meshes were still rotated incorrectly when adding the RobotModel display type.

**Comment by civerachb-cpr** [comment]
(I just received a report from a co-worker that Rviz 1.19.21 appears to render properly; the issue seems to be specifically with the 1.19.22 release. Hopefully that helps narrow it down.)

**Comment by rhaschke** [comment]
Thanks for reporting! I will have a look asap.

**Comment by rhaschke** [comment]
@clalancette + @sloretz: Please hold back any planned release sync (if any) due to this regression.

**Comment by rhaschke** [comment]
The culprit is probably https://github.com/ros-visualization/rviz/pull/1685. But I need to understand why it happens for your meshes and not for the ones I tested with...

**Comment by civerachb-cpr** [comment]
I'm confused too; in the example of the Warthog, the wheels appear to be oriented correctly, but the large yellow fender assemblies and the central chassis are not.

**Comment by rhaschke** [comment]
@civerachb-cpr: I filed a fixup. Could you please verify that this fixes the issue for you?

**Comment by knorth55** [comment]
I have the same issue.
![Screenshot from 2021-12-16 19-06-06](https://user-images.githubusercontent.com/9300063/146351386-56a0e06e-8089-42b8-a697-dabf8b0909a5.png)

**Comment by rhaschke** [comment]
Fixed via #1689.

**Comment by knorth55** [comment]
now `rviz` `1.13.23` is released in `ros-testing`, and I update `rviz` with `apt`.
It works well, thank you for the quick fix.

![Screenshot from 2021-12-17 14-06-57](https://user-images.githubusercontent.com/9300063/146492073-22c05f05-1989-4e9e-94f4-cc8298ce84fd.png)
